cypress/e2e/tests/migration/migration-waves/filter_apps.test.ts

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    login,
+    clickByText,
+    applySearchFilter,
+    deleteByList,
+    createMultipleBusinessServices,
+    createMultipleTags,
+    createMultipleApplicationsWithBSandTags,
+} from "../../../../utils/utils";
+import { Application } from "../../../models/migration/applicationinventory/application";
+import { Stakeholders } from "../../../models/migration/controls/stakeholders";
+import { Stakeholdergroups } from "../../../models/migration/controls/stakeholdergroups";
+import { manageApplications, button, name, clearAllFilters, SEC } from "../../../types/constants";
+import * as data from "../../../../utils/data_utils";
+import { MigrationWave } from "../../../models/migration/migration-waves/migration-wave";
+import { BusinessServices } from "../../../models/migration/controls/businessservices";
+import { Tag } from "../../../models/migration/controls/tags";
+
+let stakeHolders: Stakeholders[];
+let stakeHolderGroups: Stakeholdergroups[];
+const now = new Date();
+now.setDate(now.getDate() + 1);
+const end = new Date(now.getTime());
+
+end.setFullYear(end.getFullYear() + 1);
+let applicationsList: Array<Application> = [];
+let businessservicesList: Array<BusinessServices> = [];
+let tagList: Array<Tag> = [];
+
+//Automates Polarion MTA-354
+describe(
+    ["@tier2"],
+    "Migration waves: Filter validations on Manage applications modal",
+    function () {
+        before("Login and Create Test Data", function () {
+            login();
+
+            let businessservicesList = createMultipleBusinessServices(2);
```

Comment:
This is already defined on top

`businessservicesList = createMultipleBusinessServices(2);`